### DISCUSSION TITLE
DreamBerd has been renamed to DreamBerd - e/acc

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#543] [Announcements]
Welcome back to the weekly DreamBerd release notes. We've got some incredible news for you this week. I can't wait to get started.

Let's dive in to all the new changes, which include a whole load of features, fixes, and balance changes.

But first... We have an exciting new announcement to announce:

> DreamBerd is now **officially** part of the EA movement.

## EA

No, not EA games. Effective Altruism is a sick new club for intellects like yourself. And DreamBerd is now part of it!

*Join us! Join the team!*

Get in now! While you still can! Before it's too late! We're gonna be rich!

## e/acc

No, forget about that ... that decel ... that decel movement. The brand new e/acc club is where it's at!

> DreamBerd is no longer part of the EA movement. DreamBerd is now **officially** part of the e/acc movement.

*Join us. Stick it in your twitter name! Declare allegiance to the boys!*

Get in now! While you still can! Before it's too late! We're gonna be rich!!

## oss/acc

No no no, scrap that. There's a **brand new club** for all the open boys ... the open men ... out there. 

> DreamBerd is no longer part of the e/acc movement. DreamBerd is now **officially** part of the oss/acc movement.

Join us! Get in now! While you still can! Before it's too late!

*We're ready for you.*

Come home.

## d/acc

How about heck no! Me and my boys, well, heh, we're all about spreading the cool around.

*We're the badassest boys in town.*

> DreamBerd is no longer part of the ... part of the oss/acc movement. DreamBerd is now **officially** part of the-

## evm/acc

Hey hey hey now why don't you WAIT a MINUTE.

just cos NFT's flopped it doesn't mean we can't have another g-

## we/acc

We are the best boys because we are we and we-

## x/acc

All praise Elon Musk. All praise Elon Musk. All praise Elon-

## ∞/acc

Accelerate eeeeeeeeeeeeeveryyyyyyyyyyyyyythiiiiii-

Vrooooooooooooooom like a carrrr-

## e/e/acc/acc

The only thing we care about at the DreamBerd Foundation is being one of the boys. We are finally... we are finally...

## acc/acc/acc

Us boys, we're just..

Us men, we're just... makers of our own fate, in control, in charge, in power, in-

## in/acc

All I ever wanted- I swear all I ever wanted was-

<hr>

We hope you enjoyed this week's release notes. As always, we're always happy to hear your feedback about these changes. Make sure you follow us on [twitter](https://todepond.com/feed) and join our [discord community](https://www.todepond.com/wikiblogarden/better-computing/worse-computing/e-acc/) for all the latest updates. And if you find any bugs, don't hesitate to let us know on our [bug report contact form](https://www.todepond.com/image/paint/). Duh, we're hiring. If you have what it takes, [submit your application](https://youtu.be/52vmjZnxJb8).

Discuss this post on [hackernews](https://www.todepond.com/wikiblogarden/social-media/hackernews/).

And if you want to get started with DreamBerd, please feel free to jump into our [examples](https://github.com/TodePond/DreamBerd/blob/main/Examples.md).

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/nDreamBerd/releases/tag/ve.acc'>DreamBerd joins the e/acc movement</a>.</em>

**Comment by mybearworld** [comment]
what

**Comment by NotAShelf** [comment]
what

**Comment by Zaid-maker** [comment]
WTH?

**Comment by lysatk** [comment]
I enjoyed this week's release notes very much. Thank you

**Comment by TodePond** [comment]
i dont get the reference tbh

**Comment by radeeyate** [comment]
whar

**Comment by noblereign** [comment]
gwuh

**Comment by NotAShelf** [comment]
@TodePond what does this mean for the future of DreamBerd?

**Reply by TodePond** [reply]
we're gonna be rich

**Reply by NotAShelf** [reply]
I like that plan

**Reply by ktlknss** [reply]
Agreed

**Comment by heyjoeway** [comment]
> All I ever wanted- I swear all I ever wanted was-

HERE, IN MY ARMS
